### ISSUE TITLE
"task" param is ambiguous. What is a task?

### ISSUE THREAD

**Issue by matthew-groth** [jcgay/buildplan-maven-plugin#8]
I am new to maven nd finding this is a great tool tp help me learn fast. However, one thing is confusing me: `buildplan.tasks`.

A google search for "maven task" is turning up nothing, and it isn't in [the official glossery](https://maven.apache.org/glossary.html).

So for example one task is `clean`. Does this mean the clean lifecycle, the clean command, or the clean phase?

**Comment by jcgay** [comment]
Yeah this is a term used internally in Maven ([LifecycleExecutor.html#calculateExecutionPlan](https://maven.apache.org/ref/3.5.0/maven-core/apidocs/org/apache/maven/lifecycle/LifecycleExecutor.html#calculateExecutionPlan(org.apache.maven.execution.MavenSession,%20java.lang.String...))), it describes what you have provided on the maven command line; i.e. "clean jetty:run install".  
So it can be a lifecycle phase (`clean`, `install`) or a goal specification (`jetty:run`).

When you set `buildplan.tasks` to `clean`, it means the `clean` [lifecycle phase](https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html).

Is it clearer ?
